examples/edge_bundling.ipynb

```diff
@@ -0,0 +1,524 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from dask.distributed import Client\n",
+    "from pandas import DataFrame\n",
+    "\n",
+    "import fastparquet as fp\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from datashader.bundling import bundle, nop_bundle\n",
+    "from datashader.colors import inferno, viridis\n",
```

Comment:
Personally, I vote for `from colorcet import fire`; it's my go-to colormap and looks good with this data.